src/client/pages/RetroCertsCertificationPage/index.js

```diff
@@ -133,16 +133,16 @@ function RetroCertsCertificationPage(props) {
 
     setValidated(true);
 
+    setShowGenericValidaitonError(!isValid);
```

Comment:
if it takes more than a moment go ahead and merge this PR so we can get it in, but then can you start a new PR to fix the `Validaiton` typo? thank you!

Reply:
will do.